source/developer/dev-setup-centos-7.rst

```diff
@@ -0,0 +1,98 @@
+.. _dev-setup-centos-7:
+
+Installing Developer Components on CentOS 7
+===========================================
+
+Set up your development environment for building, running, and testing Mattermost.
+
+1. If you are developing with the Docker container, install and configure Docker CE:
+
+  a. Follow the Docker installation instructions at https://docs.docker.com/engine/installation/linux/docker-ce/centos/.
+
+  b. Edit your ``/etc/hosts`` file to include the following line:
+
+    ``127.0.0.1     dockerhost``
+
+  c. Add your username to the *docker* group so that you can run Docker without using sudo. Replace *{username}* with your username.
+
+    ``sudo gpasswd -a {username} docker``
+
+  d. Restart the Docker daemon.
+
+    ``sudo service docker restart``
+
+  e. Change your current group ID to the *docker* group.
+
+    ``newgrp docker``
+
+2. Install the development tools package and libpng12 required by pngquant(a library used by Mattermost).
+
+    a. ``sudo yum group install "Development Tools"``
+
+    b. ``sudo yum install -y libpng12``
+
+3. Download and install Go 1.8 for Linux:
+
+    a. Download the Go binary.
+
+       ``wget https://storage.googleapis.com/golang/go1.8.linux-amd64.tar.gz``
```

Comment:
wget is not available on CentOS 7 out of the box. Need to either install wget, or use curl.
```
curl -O https://storage.googleapis.com/golang/go1.8.linux-amd64.tar.gz
```

The yarn install instructions use wget, so probably best to install wget. On the other hand, the audience for this doc is developers, so if wget isn't on their system, they know what to do. It's up to you if you want to add a separate line for installing wget.